src/Backends/IF97/IF97Backend.h

```diff
@@ -44,14 +45,14 @@ class IF97Backend : public AbstractState  {
         this->_rhomass.clear();
         this->_hmass.clear();
         this->_smass.clear();
-        this->_phase = iphase_not_imposed; 
+        this->_phase = iphase_not_imposed;
         return true;
     };
 
     void set_phase() {
         double epsilon = 3.3e-5;                              // IAPWS-IF97 RMS saturated pressure inconsistency
-        if ((abs(_T - IF97::Tcrit) < epsilon/10.0) &&         //            RMS temperature inconsistency ~ epsilon/10
-            (abs(_p - IF97::Pcrit) < epsilon)) {              // within epsilon of [Tcrit,Pcrit]
+        if ((std::abs(_T - IF97::Tcrit) < epsilon/10.0) &&         //            RMS temperature inconsistency ~ epsilon/10
```

Comment:
I had to fix a couple of build errors on some platforms/compilers

Reply:
this one was "call to abs to ambiguous"

Fix build error: `src/Backends/IF97/IF97Backend.h:54:34: error: call of overloaded ‘abs(double)’ is ambiguous`

Found with the manylinux_2_24_x86_64

gcc (Debian 6.3.0-18+deb9u1) 6.3.0 20170516

Reply:
Correct solution I think